src/main/java/duke/tasklist/task_types/ToDo.java

```diff
@@ -0,0 +1,26 @@
+package duke.tasklist.task_types;
+
+/**
+ * Represents a <code>task</code> object with the todo tag into it.
+ * 
```

Comment:
Consider removing the blank line at line 5 to keep a consistent format for the header Javadoc of each class.